fontir/src/orchestration.rs

```diff
@@ -30,6 +30,52 @@ bitflags! {
     }
 }
 
+/// Generates fn $getter_name(&self) -> Arc<$value_type>.
+///
+/// Assumes we are in an impl block for a Context and that
+/// self.$lock_name is an Arc<RwLock<Option<Arc<$ir_type>>>>
```

Comment:
If we're always using `Arc<RwLock<Option<Arc<T>>>>` it's probably worth having some wrapper type to hide the mess, like `Resource<T>` or something?

Reply:
It is ... often but not always the case. I suppose it's consistent for everything *except* glyphs, which is probably enough to justify the type. Will add.